service/awsconfig/v10/cluster_resource_set.go

```diff
@@ -263,6 +263,7 @@ func NewClusterResourceSet(config ClusterResourceSetConfig) (*framework.Resource
 				KMS:            config.GuestAWSClients.KMS,
 				ELB:            config.GuestAWSClients.ELB,
 			},
+			EBS: ebsService,
```

Comment:
The `cloudformation` resource is now also configured with the EBS service in order to shutdown the master and detach the etcd volume before we update the guest's main stack.